packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -78,6 +83,8 @@ class AttestationRequestHandler {
       logging: this.sequelizeLogger,
     })
 
+    await this.verifyPepperIfApplicable()
```

Comment:
If this doesn't need to be async maybe we should move it in front of findAttestationByKey so we fail faster?